js/src/components/google-combo-account-card/connect-ads/connect-ads-body.js

```diff
@@ -0,0 +1,84 @@
+/**
+ * External dependencies
+ */
+import { useEffect, useRef } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import AdsAccountSelectControl from '.~/components/ads-account-select-control';
+import AppButton from '.~/components/app-button';
+import ContentButtonLayout from '.~/components/content-button-layout';
+import ConnectedIconLabel from '.~/components/connected-icon-label';
+import LoadingLabel from '.~/components/loading-label/loading-label';
+import useEventPropertiesFilter from '.~/hooks/useEventPropertiesFilter';
+import { FILTER_ONBOARDING } from '.~/utils/tracks';
+
+/**
+ *
+ * @param {Object} props Props.
+ * @param {Array} props.accounts Accounts.
+ * @param {Object} props.googleAdsAccount Google Ads account object.
+ * @param {boolean} props.isConnected Whether the account is connected.
+ * @param {Function} props.handleConnectClick Callback to handle the connect click.
+ * @param {boolean} props.isLoading Whether the card is in a loading state.
+ * @param {Function} props.setValue Callback to set the value.
+ * @param {string} props.value Ads account ID.
+ * @return {JSX.Element} Body component.
+ */
+const ConnectAdsBody = ( {
+	accounts,
+	googleAdsAccount,
+	isConnected,
+	handleConnectClick,
+	isLoading,
+	setValue,
+	value,
+} ) => {
+	const initialValueRef = useRef( googleAdsAccount.id );
+	const getEventProps = useEventPropertiesFilter( FILTER_ONBOARDING );
+
+	useEffect( () => {
+		setValue( initialValueRef.current );
+	}, [ initialValueRef, setValue ] );
+
+	const ConnectCTA = () => {
+		if ( isConnected && initialValueRef.current === Number( value ) ) {
```

Comment:
@asvinb It checks whether:

1. Ads account is connected.
2. The currently selected ads account is the same which is connected.

In this case, we show `Connected` label instead of `Connect` button.

Reply:
@ankitrox Not sure if am following here. When in connected state, the select is disabled, meaning the only way to change the Ads ID is by disconnecting the Ads account.
https://www.figma.com/design/fqR0EHi63lWahRcVTKCcba/Google-Listings-%26-Ads-v2.x?node-id=7662-6154&node-type=frame&t=FZW0ZlwmMV6RKllP-0

So if we just check for `isConnected` is fine.